Assets/_BForBoss/_Core/Scripts/Labor/RingSystem/RingLaborManager.cs

```diff
@@ -29,22 +27,31 @@ public void Reset()
             _laborSystem = new LaborSystem(_ringSystems);
         }
         
-        public void Initialize()
+        public void Initialize(CountdownViewBehaviour countdownView)
```

Comment:
`CountdownViewBehaviour` 

Make this a singleton, or just call its static shared instance.

Reply:
Split countdown view into a singleton countdown timer, and turned the view into more of a view for that